paddle/fluid/operators/metrics/precision_recall_op.cc

```diff
@@ -22,57 +22,73 @@ class PrecisionRecallOp : public framework::OperatorWithKernel {
   using framework::OperatorWithKernel::OperatorWithKernel;
 
   void InferShape(framework::InferShapeContext *ctx) const override {
-    PADDLE_ENFORCE(ctx->HasInput("MaxProbs"),
-                   "Input(MaxProbs) should not be null.");
-    PADDLE_ENFORCE(ctx->HasInput("Indices"),
-                   "Input(Indices) should not be null.");
-    PADDLE_ENFORCE(ctx->HasInput("Labels"),
-                   "Input(Labels) should not be null.");
-    PADDLE_ENFORCE(ctx->HasOutput("BatchMetrics"),
-                   "Output(BatchMetrics) should not be null.");
-    PADDLE_ENFORCE(ctx->HasOutput("AccumMetrics"),
-                   "Output(AccumMetrics) should not be null.");
-    PADDLE_ENFORCE(ctx->HasOutput("AccumStatesInfo"),
-                   "Output(AccumStatesInfo) should not be null.");
+    PADDLE_ENFORCE_EQ(ctx->HasInput("MaxProbs"), true,
+                      platform::errors::InvalidArgument(
```

Comment:
`InvalidArgument` -> `NotFound`, or use MACRO `Op_INOUT_CHECK` , same for others